cyberfusion/NextCloudSupport/__init__.py,sha256=8JByOElHmHLFiNx9xTHIU6dy5AjPQQmpPLFq5IR4bpM,46
cyberfusion/NextCloudSupport/_occ.py,sha256=46gDht7KKwNlDJ46YUrDdm4PQxYV0JtlzLyOTYx_ZGE,908
cyberfusion/NextCloudSupport/app.py,sha256=IIH7N9F8i-vFnajXxw3BJO_5i9U5VxQ7rOqtcWqo-Q0,3901
cyberfusion/NextCloudSupport/exceptions.py,sha256=--rrdtxwwCUmuukRV7sNIIughMxRHv9dD3s_p2lTtNY,569
cyberfusion/NextCloudSupport/instance.py,sha256=oCuoyvKEJ7pbOV5oQE1ad5aK1kOSKCptwaydVW67L-s,9351
cyberfusion/NextCloudSupport/user.py,sha256=CSSBzONswPu6IfCtl3vQ3dsLxpeF97bLvBcyeqTuCPw,375
python3_cyberfusion_nextcloud_support-1.1.1.2.2.dist-info/METADATA,sha256=cX4nwx6ifGTtemnx3FLlK4R5BkuL3jOzgezB_SzJhT0,945
python3_cyberfusion_nextcloud_support-1.1.1.2.2.dist-info/WHEEL,sha256=Mdi9PDNwEZptOjTlUcAth7XJDFtKrHYaQMPulZeBCiQ,91
python3_cyberfusion_nextcloud_support-1.1.1.2.2.dist-info/top_level.txt,sha256=ss011q9S6SL_KIIyq7iujFmIYa0grSjlnInO7cDkeag,12
python3_cyberfusion_nextcloud_support-1.1.1.2.2.dist-info/RECORD,,
